src/nodedef.cpp

```diff
@@ -112,6 +183,71 @@ void NodeBox::deSerialize(std::istream &is)
 		wall_side.MinEdge = readV3F1000(is);
 		wall_side.MaxEdge = readV3F1000(is);
 	}
+	else if (type == NODEBOX_CONNECTED)
+	{
+		u16 count = readU16(is);
+		fixed.reserve(count);
+		while (count--) {
+			aabb3f box;
```

Comment:
can you check, if i remember aabb3f has a constructor. Use if and push the variable directly in the push_back if the constructor exists